environments/sre/servicemonitors/observatorium-thanos-querier-production.servicemonitor.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    prometheus: app-sre
+  name: observatorium-thanos-querier
```

Comment:
I realized that since these are landing in the same namespace in the end, we'd need to suffix the `name` field with `-stage` and `production`